src/Components/Facility/FacilityHome.tsx

```diff
@@ -80,76 +73,99 @@ export const FacilityHome = (props: any) => {
 
   useMessageListener((data) => console.log(data));
 
-  const fetchData = useCallback(
-    async (status: statusType) => {
+  const { res: permittedFacilityRes, data: permittedFacilityData } = useQuery(
```

Comment:
If you have multiple queries in the same component, consider doing this instead.
you can access res and data by: `facilityQuery.res / facilityQuery.data` which is much cleaner right?

```suggestion
  const facilityQuery = useQuery(
```